src/main/cliapp/src/containers/geneGeneticInteractionsPage/GeneGeneticInteractionsTable.js

```diff
@@ -0,0 +1,223 @@
+import React, { useRef, useState } from 'react';
+
+import { GenericDataTable } from '../../components/GenericDataTable/GenericDataTable';
+import { IdTemplate } from '../../components/Templates/IdTemplate'; 
+import { GenomicEntityTemplate } from '../../components/Templates/genomicEntity/GenomicEntityTemplate';
+import { BooleanTemplate } from '../../components/Templates/BooleanTemplate';
+import { getDefaultTableState } from '../../service/TableStateService';
+import { FILTER_CONFIGS } from '../../constants/FilterFields';
+import { evidenceTemplate } from '../../components/EvidenceComponent';
+import { OntologyTermTemplate } from '../../components/Templates/OntologyTermTemplate';
+import { ListTableCell } from '../../components/ListTableCell';
+import { EllipsisTableCell } from '../../components/EllipsisTableCell';
+import { Tooltip } from 'primereact/tooltip';
+
+export const GeneGeneticInteractionsTable = () => {
+
+	const [isInEditMode, setIsInEditMode] = useState(false); //needs better name
+	const [errorMessages, setErrorMessages] = useState({});
+	const errorMessagesRef = useRef();
+	errorMessagesRef.current = errorMessages;
+
+	const [uiErrorMessages, setUiErrorMessages] = useState([]);
+	const uiErrorMessagesRef = useRef();
+	uiErrorMessagesRef.current = uiErrorMessages;
+
+	const toast_topleft = useRef(null);
+	const toast_topright = useRef(null);
+
+	const sortMapping = {
+		'geneAssociationSubject.geneSymbol.displayText': ['geneAssociationSubject.geneFullName.displayText', 'geneAssociationSubject.modEntityId'],
+		'geneGeneAssociationObject.geneSymbol.displayText': ['geneGeneAssociationObject.geneFullName.displayText', 'geneGeneAssociationObject.modEntityId'],
+		'phenotypeAnnotationObject.name': ['phenotypeAnnotationObject.curie', 'phenotypeAnnotationObject.namespace'],
+		'interactionType.name': ['interactionType.curie'],
+		'interactorARole.name': ['interactorARole.curie'],
+		'interactorBRole.name': ['interactorBRole.curie'],
+		'interactorAType.name': ['interactorAType.curie'],
+		'interactorBType.name': ['interactorBType.curie'],
+		'interactionSource.name': ['interactionSource.curie'],
+		'interactorAGeneticPerturbation.alleleSymbolDisplayText': ['interactorAGeneticPerturbation.alleleFullName.displayText', 'interactorAGeneticPerturbation.modEntityId'],
+		'interactorBGeneticPerturbation.alleleSymbolDisplayText': ['interactorBGeneticPerturbation.alleleFullName.displayText', 'interactorBGeneticPerturbation.modEntityId']
+	};
+
+	const phenotypesOrTraitsBodyTemplate = (rowData) => {
```

Comment:
This could be pulled out into a template component. I would look somewhat similar to the `EvidenceCodesTemplate`.

Reply:
@adamgibs Good point - done